tests/integration.coffee

```diff
@@ -946,8 +946,12 @@ describe 'SDK Integration Tests', ->
 
 					describe 'resin.models.device.getDeviceUrl()', ->
 
+						it 'should eventually be a url containing full length uuid', ->
```

Comment:
Unsure if this should just be included in the following `'should eventually be an absolute url'` test or it's own? @jviotti